### PR TITLE
[Feature] Added the missing Arabic translations

### PR DESCRIPTION
**Pull request by arsangamal**
## What does this PR do?

Added the Arabic missing translations in `app/config/locale/translations/ar.json`

## Test Plan

Tested the Email templates to get the right translations

## Related PRs and Issues

[Help improve Appwrite's Arabic language translation](https://github.com/appwrite/appwrite/issues/1673)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes, carefully

### CODE REVIEW

File: app/config/locale/translations/ar.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "تأكيد الحساب",
+    "emails.verification.hello": "مرحبا {{name}}",
+    "emails.verification.body": "برجاء اتباع الرابط التالي لتأكيد بريدك الإلكتروني",
+    "emails.verification.footer": "لو لم تطلب تأكيد هذا البريد الإلكتروني ، يمكنك تجاهل هذه الرسالة",
```
Comment (YazeedAlKhalaf) [inline app/config/locale/translations/ar.json:9]: `الإلكتروني ، يمكنك` should be `الإلكتروني، يمكنك`

**Review by YazeedAlKhalaf — APPROVED** [review]
Awesome work! 🚀

File: app/config/locale/translations/ar.json (hunk 1 above)
Comment (arsangamal) [reply app/config/locale/translations/ar.json]: Yea, fixed at the same PR.
Thank you @YazeedAlKhalaf

**Comment by christyjacob4** [comment]
Thank you so much for your help @YazeedAlKhalaf  @arsangamal :pray:

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU so much @arsangamal  ! All changes merged 🥳
Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**